tests/integration/security/testdata/automtls-partial-sidecar-dr-disable.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
+metadata:
+  name: "default"
+  annotations:
+    test-suite: "automtls-partial-dr-disable"
+spec:
+  mtls:
+    mode: STRICT
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: "multiversion-route"
+  annotations:
+    test-suite: "automtls-partial-dr-disable"
+spec:
+  hosts:
+  - "multiversion"
+  http:
+  - name: "vistio-route"
+    match:
+    - uri:
+        prefix: "/vistio"
+    route:
+    - destination:
+        host: "multiversion"
+        subset: "vistio"
+  - name: "legacy-by-default"
+    route:
```

Comment:
match "/vlegacy" path here?

Reply:
my intention is this will become default. but either way, this achieve the affect that "/vlegacy" was routed to the no sidecar deployments.

Reply:
since the /random path is not exercised in the test.